src/view/view.rs

```diff
@@ -78,10 +96,14 @@ pub trait View<T, A = ()>: ViewMarker + Send {
     ) -> MessageResult<A>;
 }
 
+pub trait View<T, A = ()>: GenericView<T, WidgetBound, A> {}
+
+impl<T, A, V: GenericView<T, WidgetBound, A>> View<T, A> for V {}
+
```

Comment:
Is the idea here that `View` is vello-renderer (=xilem widgets) specific, and `GenericView` could apply to other renderers such as one targetting web, svg, etc? (and that we might therefore get an `SvgView` trait similar to `View` as part of such a backend?)

If so, I'd be tempted to call these `View` and `NativeView` rather than `GenericView` and `View`. But that's probably not important (and could always be changed later). Perhaps just a little note on `View` explaining the difference between it and `GenericView` would be good for now?